### PR TITLE
Feature7/forms

### PR DESCRIPTION
**Pull request by techEmprez**
### Project requirements
-  Create a method called `current_user` in your ApplicationController to make current user data available to all controllers.
    -  It will return the first user from the database.
    -  Don't worry about proper authentication at this time. You will add it in the upcoming projects.
- Create forms to perform the following functions:
    - Creates a Post on behalf of the `current_user` (use the method that you created in your ApplicationController).
    - Create a comment on behalf of the `current_user` (use the method that you created in your ApplicationController).
- Allow Users to add likes to Posts.

### CODE REVIEW

File: app/views/posts/show.html.erb [hunk 1]
```diff
@@ -14,3 +14,8 @@
         <span> <%= comment.author.name %>: <%= comment.text%> </span>
     <% end %>
 </div>
+
+
+<div class="mx-auto" style="width: 200px;">
+<%=link_to "New Comment", new_user_post_comment_path(@user.id, @post.id), class:"btn btn-dark mb-4" %>
+</div>
```
Comment (omarsalem7) [inline app/views/posts/show.html.erb]: - [ ] Kindly consider the user should be able to add a like to posts by adding a button for like under the post.

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @techEmprez 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- The Pull Request has a proper title and description. ✔️
- Correct git-flow ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/posts/show.html.erb (hunk 1 above)
Comment (techEmprez) [reply app/views/posts/show.html.erb]: Done

File: app/views/comments/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<div>
+    <h2> Create Comment </h2>
+    <%= form_with model: @comment, url: user_post_comments_path, data: { turbo: false} do |form| %>
+        
+        <div class="mb-3">
+            <%= form.label :text, class: "form-label" %>
+            <%= form.text_field :text, class: "form-control", id: "exampleFormControlInput1", placeholder: "Post Title" %>
+        </div>
```
Comment (mirouhml) [inline app/views/comments/new.html.erb]: [OPTIONAL]

- [ ] You forgot to change the placeholder here, kindly change it.

**Review by mirouhml — APPROVED** [review]
# APPROVED 🟢

Hi @techEmprez, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- Great job setting up the correct linters ✔️
- Highly organized and clean code ✔️ 👨🏻‍💻
- Professional documentation ✔️
- Correct workflow ✔️
- Meaningful commit messages ✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/comments/new.html.erb (hunk 2 above)
Comment (techEmprez) [reply app/views/comments/new.html.erb]: Done

**Pull request merged**